libraries/rush-lib/src/logic/base/BaseInstallManager.ts

```diff
@@ -160,16 +163,23 @@ export abstract class BaseInstallManager {
       subspace,
       { npmrcHash: npmrcHash || '<NO NPMRC>' }
     );
+    if (isFilteredInstall) {
+      // Get the projects involved in this filtered install
+      const selectedProjectNames: string[] = this.options.pnpmFilterArguments.filter(
```

Comment:
Hmm. Parsing this out of the args feels a bit janky. Maybe we should update the options to contain the set of projects to include, and we would create the PNPM filter from it before running the install below?

Reply:
We should also link to the filter spec wherever we construct the filter

Reply:
Also, I think it's the case (especially given the below code), but confirm that we include all in-scope packages, and aren't just constructing the filter using `@project/name...`.

Reply:
This is a good suggestion 👍

Reply:
Refactored to only include the project names in the options, then adding the "--filter" tag in the WorkspaceInstallManager instead